Sources/Testing/Events/Clock.swift

```diff
@@ -93,6 +93,22 @@ extension Test.Clock.Instant {
   public var durationSince1970: Duration {
     Duration(wall)
   }
+
+  /// Get the number of nanoseconds from this instance to another.
+  ///
+  /// - Parameters:
+  ///   - other: The later instant.
+  ///
+  /// - Returns: The number of nanoseconds between `self` and `other`. If
+  ///   `other` is ordered before this instance, the result is negative.
+  func nanoseconds(until other: Test.Clock.Instant) -> Int64 {
```

Comment:
Can we add (or do we already have) a unit test for `nanoseconds(until:)`?

Reply:
We have indirect test coverage for it already, yeah, but I can add an explicit parameterized test.

Reply:
We have indirect code coverage, but I can add a parameterized test for it directly.